components/director/internal/domain/formation/service.go

```diff
@@ -362,6 +364,12 @@ func (s *service) assign(ctx context.Context, tnt, objectID string, objectType g
 	if err != nil {
 		return nil, errors.Wrapf(err, "while getting formation %q", formation.Name)
 	}
+	// TODO Remove default scenario check after removing default scenario
```

Comment:
Is this comment referring to the current PR or a future improvement?

Reply:
The comment is for when the default scenario is removed in the future.